versions/1.0/SPEC.md

```diff
@@ -1036,6 +1037,42 @@ output {
 }
 ```
 
+#### Files and Optional Outputs
+
+File outputs are described as string paths relative to the initial working
```

Comment:
Depeding on what you mean by `initial working directory`, this might conlict with another part of the spec:
At [File Outputs](https://github.com/openwdl/wdl/blob/master/versions/development/SPEC.md#file-outputs):
```
Individual File outputs can be declared using a String path relative to the execution directory. For example:
```

Here I'm assuming that the `initial working directory` refers to the directory one runs their execution engine from, whereas `execution directory` refers to a dedicated workspace for each task (ie. the cromwell implementation).

I'm wondering how making paths relative to the initial working directory will affect things like globbing and scattering. This might greatly the risk of tasks overwritting each others outputs in the case of scatters and with globbing the globbed files may end up including those from multiple tasks (or even completely unrelated files, if you already had  files in the working directory to begin with).